### ISSUE TITLE
binary operators

### ISSUE THREAD

**Issue by luciusmagn** [jonathandturner/rhai#24]
Implement the missing binary operators:
```rust
let rem = 10 % 4; // 2
let lsh = 10 << 4; // 160
let rsh = 10 >> 4; // 0
let and = 10 & 4; // 0
let or = 10 | 4; // 14
let xor = 10 ^ 4; // 14
```

**Comment by luciusmagn** [comment]
#42 implements all but modulo

**Comment by luciusmagn** [comment]
implemented in #47
